Dockerfile

```diff
@@ -31,6 +31,7 @@ COPY . /usr/src/app
 
 RUN git clone https://github.com/smart-on-fhir/client-py validator && \
     cd validator && \
-    git reset --hard fd67e6a8717edfc63fb7293405a1c45a4cfac8c2
+    git reset --hard fd67e6a8717edfc63fb7293405a1c45a4cfac8c2 && \
+    touch __init__.py
```

Comment:
Thanks! Does pylint run as part of the travis tests BTW? I don't remember this generating an error (but I may have just been oblivious).